test/artefacts/register-a-death/overseas/north-korea.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
```

Comment:
@floehopper @chrisroos would you know why are we getting changes in North-Korea artefacts if nothing related to this country has been modified in this PR? Is there any document, or any point of the codebase we can look into?